cookbook/Berksfile

```diff
@@ -0,0 +1,8 @@
+source 'https://supermarket.chef.io'
+
+metadata
+
+cookbook 'java'
+cookbook 'et_nginx'
```

Comment:
should this just be the regular nginx cookbook, looks like this might be evertrue specific?